code/game/atoms_movable.dm

```diff
@@ -7,6 +7,8 @@
 	var/move_resist = MOVE_RESIST_DEFAULT
 	var/move_force = MOVE_FORCE_DEFAULT
 	var/pull_force = PULL_FORCE_DEFAULT
+	/// bigger value slower pulling speed
+	var/pull_speed = 0 
```

Comment:
Is there any reason for this variable to be on `/atom/movable`? Since this PR doesnt directly add slowdown to pulling mobs, this should probably be moved to `/obj`

Reply:
`pulling` is scoped as `atom/moveable` on `atom/moveable`. Scoping it on `obj` will require typecasting it in the `human_movement` code.

Reply:
As much as I don't like adding variables to very low level types, I will agree that a pull slowdown should be scoped to the same level that pulling is defined on. Even though this really effects objects right now, it's quite reasonable for someone to apply this to mobs in they chose to do so in the future.

Reply:
i wonder if i should fix this or just close the PR, people seem to not like this 
![image](https://github.com/ParadiseSS13/Paradise/assets/61080616/6b5bf61f-ec24-4f5c-82d5-06745540f2ae)

Reply:
that is up to you I guess. Github reactions aren't really a metric on how the community feels about a change, just how the most active github users are thinking at the moment (most dev team members don't even react to PRs either). Atm there's currently 3 votes in favor of this and 1 against on the balance team.

Reply:
alright then, moved variable to /obj, i guess thats fine now?
https://github.com/ParadiseSS13/Paradise/pull/21056/commits/e2627b584169ed0af8fb6ac31b5b3429a920c776